packages/block-editor/src/components/inspector-controls/groups.js

```diff
@@ -5,10 +5,14 @@ import { createSlotFill } from '@wordpress/components';
 
 const InspectorControlsDefault = createSlotFill( 'InspectorControls' );
 const InspectorControlsAdvanced = createSlotFill( 'InspectorAdvancedControls' );
+const InspectorControlsDimensions = createSlotFill(
+	'InspectorDimensionsControls'
```

Comment:
We probably could use the following form to align with `BlockControls`:
```suggestion
	'InspectorControlsDimensions'
```

<img width="1240" alt="Screen Shot 2021-08-24 at 09 55 07" src="https://user-images.githubusercontent.com/699132/130578904-aeef7eb4-97df-407a-8008-4622a5c9811b.png">

The existing exception are mostly for backward-compatibility like `BlockFormatControls` or `InspectorAdvancedControls`. Well, for everything else that was never used in the code as a string we can go with a consistent pattern we agree upon.

Reply:
Sounds good to me.

✅  Done.